src/ert/_c_wrappers/util/substitution_list.py

```diff
@@ -109,8 +109,8 @@ def indexForKey(self, key):
 
         return None  # Should never happen!
 
-    def substitute(self, to_substitute: str) -> str:
-        return self._alloc_filtered_string(to_substitute)
+    def substitute(self, to_substitute: str, context: str) -> str:
```

Comment:
Probably want the context to default to empty string here.

Reply:
done, and handled on C side so we don't get something like
```
... - context was ``
```